include/crm/common/xml.h

```diff
@@ -301,6 +301,29 @@ int xml_apply_patchset(xmlNode *xml, xmlNode *patchset, bool check_version);
 
 void patchset_process_digest(xmlNode *patch, xmlNode *source, xmlNode *target, bool with_digest);
 
+enum acl_ns {
+    acl_ns_writable = 0x01,
+    acl_ns_readable = 0x02,
+    acl_ns_denied = 0x04,
+};
```

Comment:
Wouldn't it make sense to reuse the xpf_acl flags?

Reply:
Could be, but at the cost of publicizing them, which may not be a good
move :-/

Reply:
Oh right, never mind